tendrl/commons/jobs/__init__.py

```diff
@@ -126,14 +126,21 @@ def _run(self):
                             Event(
                                 ExceptionMessage(
                                     priority="error",
-                                    job_id=job.job_id,
```

Comment:
ExceptionMessage is a subclass of Message, it can take job_id and flow_id as input